src/controllers/entities.js

```diff
@@ -1,17 +1,16 @@
 const HAPIRestAPI = require('@envage/hapi-pg-rest-api')
 const Joi = require('joi')
-const { get } = require('lodash')
 const { version } = require('../../config')
 const { pool } = require('../lib/connectors/db')
-const { groupBy, reduce } = require('lodash')
 const Boom = require('@hapi/boom')
 
 const isIndividual = entity => {
-  return get(entity, 'entity_type', '').toLowerCase() === 'individual'
+  const entityType = entity.entity_type ? entity.entity_type : ''
+  return entityType.toLowerCase() === 'individual'
 }
 
 const lowerCaseEntityName = entity => {
-  const name = get(entity, 'entity_nm', '')
+  const name = entity.entity_nm ? entity.entity_nm : ''
```

Comment:
Another place we can use nullish coalescing
```suggestion
  const name = entity.entity_nm ?? ''
```